### DISCUSSION TITLE
Not able to install home brew on the macbook air version 11.6

### DISCUSSION THREAD

**Discussion by ArifaMujawarSM** [Homebrew/discussions#2343] [Everyday usage]
while installing homebrew...getting below error
<img width="1440" alt="Screenshot 2021-10-24 at 1 46 56 PM" src="https://user-images.githubusercontent.com/84315899/138605950-33a1434d-90af-4106-b532-e90288b9d325.png">

**Comment by carlocab** [comment]
Please [don't use screenshots](https://meta.stackoverflow.com/a/285557) when sharing the output of commands in your terminal. They're often difficult to read and impossible to reuse. Instead, copy the text here and wrap them in three back-ticks (```) on the line before and on the line after to format them properly.

From what I can tell (though it's not very clear), this looks like an issue with your machine. Try reading through #666 for solutions others have had with similar problems.
